dune-project

```diff
@@ -2,6 +2,9 @@
 (name coq)
 (using coq 0.1)
 
+(formatting
+ (enabled_for ocaml))
```

Comment:
This is useless in Dune 2.0. Formatting is enabled by default.

Reply:
This actually disables formatting for `dune` files, otherwise those will get reformatted as well. If we want, we should do that in a separate PR, this is about OCaml.

Reply:
Hum, yeah I think we want this. I don't see the need to make this a separate PR but fine.

Reply:
Well auto-formatting of dune files is fairly orthogonal to auto-formatting of OCaml files, much less choice there :)

Reply:
Not orthogonal in the implementation, since you had to add these two lines :laughing: